Seohee/Lv2/메뉴 리뉴얼.js

```diff
@@ -0,0 +1,68 @@
+// 배열과 조합의 수를 인자로 전달받으며
+function getCombinations(arr, selectNumber) {
+    let results = [];
+    // 만약, 조합의 수가 1이라면 재귀함수를 종료하게 되어있다
+    if (selectNumber === 1) return arr;
+
+    // 배열 각각에 대한 원소를 순회한다
+    arr.forEach((elem, index, origin) => {
+        const rest = origin.slice(index + 1);
+        // 그 나머지 배열과 조합의수 -1의 값을 다시 재귀함수로 호출
+        const combinations = getCombinations(rest, selectNumber - 1);
+        const attached = combinations.map((combination) => [elem, ...combination]);
+        results.push(...attached);
+    })
+    results = results.map(result => {
+        result.sort();
+        return result.join('');
+    })
+    return results;
```

Comment:
```suggestion
    return results.map(result => result.sort().join(''));
```
이 경우에는 이렇게 표현하는 방법도 있을거 같아요